### PR TITLE
add cmd impl of change-advisory-status

### PR DESCRIPTION
**Pull request by rioliu-rh**
- add cmd implementation of `change-advisory-status`

- command help
```
oar -r 4.12.19 change-advisory-status -h
Usage: oar change-advisory-status [OPTIONS]

  Change advisory status e.g. QE, REL_PREP

Options:
  -s, --status TEXT  Valid advisory status, default is REL_PREP
  -h, --help         Show this message and exit.
```
- command usage
```
oar -r 4.12.19 change-advisory-status -s REL_PREP
```

### CODE REVIEW

File: oar/cli/cmd_change_advisory_status.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+@click.pass_context
+@click.option(
+    "-s",
+    "--status",
+    default=AD_STATUS_REL_PREP,
+    help="Valid advisory status, default is REL_PREP",
+)
+def change_advisory_status(ctx, status):
+    """
+    Change advisory status e.g. QE, REL_PREP
```
Comment (wewang58) [inline oar/cli/cmd_change_advisory_status.py:21]: seems qe cannot change advisory to QE,  Just move to REL_PREP

Comment (rioliu-rh) [reply oar/cli/cmd_change_advisory_status.py:22]: it does not matter. the func is just used to change the status. the cmd can be used to change advisory status from `NEW_FILES` to `QE` as well.

**Comment by rioliu-rh** [comment]
/retitle OCPQE-17509 add cmd impl of change-advisory-status

**Pull request merged**